backend/worker_handler/permissions/admin_permissions_inst_hierarchy_handler.py

```diff
@@ -0,0 +1,70 @@
+from permissions import DEFAULT_ADMIN_PERMISSIONS
+from worker_handler import get_all_parent_admins
+from worker_handler import add_permission_to_children
+from worker_handler import is_not_admin
+from worker_handler import should_remove
+from worker_handler import BaseHandler
+from google.appengine.ext import ndb
+from util import NotificationsQueueManager
+
+
+__all__ = ['RemoveAdminPermissionsInInstitutionHierarchyHandler', 'AddAdminPermissionsInInstitutionHierarchyHandler']
+
+class RemoveAdminPermissionsInInstitutionHierarchyHandler(BaseHandler):
```

Comment:
I think a better name for this file would be 'manage_admin_permission_handler.py'